scraper/nginx_plus.go

```diff
@@ -120,14 +121,25 @@ func (scr *NginxPlusScraper) scrapeUpstream(status *Status, metrics chan<- metri
 		}
 
 		for _, peer := range upstream.Peers {
+			if len(scr.excludeUpstreamAddresses) != 0 {
+				exclude := false
+				for _, address := range scr.excludeUpstreamAddresses {
```

Comment:
It seems that map[addr]bool suits better here

Reply:
Good idea.